src/arkade-chainswap.ts

```
@@ -0,0 +1,1362 @@
+                    case "transaction.confirmed":
+                        await updateSwapStatus();
+                        break;
+                    case "transaction.server.mempool":
+                    case "transaction.server.confirmed":
+                        if (!data.transaction?.hex) {
+                            await updateSwapStatus();
+                            break;
+                        }
+                        const updatedSwap = await updateSwapStatus(
+                            data.transaction.hex
+                        );
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)